### ISSUE TITLE
Failed to initialize LBP port

### ISSUE THREAD

**Issue by akiskourtis** [open-ness/specs#42]
Hello all,

I have managed to built the on premises setup on a set of servers, and also succeeded to setup a connection on a LTE system, using the edge node as a bridge. However, when I try to add the LBP port the process fails to enable and the nts container fails.

I manage to restart it, and attach to it and the error that it ouputs for the LBP port is:
**NES: [ERR] Missing: section PORT0, entry lbp-mac, in config file.
NES: [ERR] Failed to add dataplane rules.
NES: [ERR] Could not initialize nes ACL.
NES: [ERR] Could not initialize ctrl.**

Am I missing something in the configuration, that i should add? the error points somewhere that the mac of the lbp port is missing from somewhere. Is it missing from the portal or i should add it somewhere else also?

Note: that the UE/eNB port and the EPC port are pinging just fine, and the S1 establishment is being done properly.

Thank you

**Comment by krishnajs** [comment]
@akiskourtis have you update the NTS config file with the required LBP ports?  "NES: [ERR] Missing: section PORT0, entry lbp-mac, in config file." suggests missing information. Maybe you can also share the config file.

**Comment by akiskourtis** [comment]
I see that if i do it manually (add the lbp-mac in the nts.cfg file), the docker starts properly (the traffic does not arrive a the lbp, but maybe that's another issue)
But if I do it through the portal, there is not way, at least from what I see to input the lbp-mac field.

My sample cfg file

[PORT0]
name = 0000:01:00.3
description = I350 Gigabit Network Connection
pci-address = 0000:01:00.3
traffic-type = IP
traffic-direction = lbp
egress-port = 2
MTU = 1500

[PORT1]
name = 0000:81:00.0
description = [enp129s0f0] Ethernet Controller 10-Gigabit X540-AT2
pci-address = 0000:81:00.0
traffic-type = IP
traffic-direction = upstream
egress-port = 2
mac = a0:36:9f:69:6c:f8
MTU = 1500
route = prio:99,ue_ip:0.0.0.0/0,srv_ip:192.168.100.2/32,encap_proto:noencap

[PORT2]
name = 0000:81:00.1
description = Ethernet Controller 10-Gigabit X540-AT2
pci-address = 0000:81:00.1
traffic-type = IP
traffic-direction = downstream
egress-port = 1
MTU = 1500

[VM common]
max = 32
number = 2
vhost-dev = /var/lib/appliance/nts/qemu/usvhost-1

[NES_SERVER]
ctrl_socket = /var/lib/appliance/nts/control-socket

[KNI]
max = 32

**Comment by akiskourtis** [comment]
side note: in the policy editor if i don't fill the source IP it does not let me save and apply it, so I put it as 0.0.0.0, is it correct?

**Comment by akiskourtis** [comment]
Apparently, I did not assign the policy on the correct interface, and also the timeline of the actions plays an important role.
I noticed that after I define the breakout interface, then I need to also add the policy (from there it gets the lbp-mac field)

Just one minor comment for the documentation, the 0.0.0.0 parameter on the input part of the policy needs to be added (in the comments, or the snapshot), as it may be confusing at the start of the deployment.

Apart from that I believe the issue can be closed, as it can be solved either manually adding the lbp-mac on the nts.cfg, or carefully updating the UI parameters.

Thank you for the support @krishnajs !
